dev-support/hbase-personality.sh

```diff
@@ -513,6 +516,7 @@ function hadoopcheck_parse_args
 ## @stability    evolving
 function hadoopcheck_docker_support
 {
+  DOCKER_EXTRAARGS=("${DOCKER_EXTRAARGS[@]}" "--ulimit" "nproc=12500")
```

Comment:
I don't think this should be here. we want this proc limit change all the time, not just when the hadoopcheck test is enabled.

Reply:
I don't follow. The new 12500 number is put everywhere, not just here conditional on when hadoopcheck runs. Thanks for taking a look.

Reply:
the reference you've added here is in the function that defines what is needed when we run docker specifically _just_ when we're doing the hadoopcheck test.

if we need it all the time, then it should be elsewhere. It looks it should be coming in from the YETUS_ARGS that have been added to the wrapper. If that's the case, then removing it here should have no impact on what happens. If removing it here makes it so things don't do what you expect, then something else is broken because the inclusion here is only going to happen when the hadoopcheck test is enabled.

Reply:
Let me push new patch w/ your change. Thanks @busbey